grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -90,6 +232,7 @@ class GrabbitConfigurationSpec extends Specification {
                     },
                     {
                         "path" : "/content/dam/d/images",
+                        "excludePaths" : [],
```

Comment:
Why are all these
```groovy
"excludePaths" : [],
```
lines here? Doesn't the JSON work if it's omitted? It seems like it's just added a lot of cruft/complexity to the spec with no benefit.